packages/web-components/src/components/ic-checkbox-group/ic-checkbox-group.stories.js

```diff
@@ -315,6 +315,26 @@ export const ConditionalStatic = {
   name: "Conditional static",
 };
 
+export const SlottedLabel = {
+  render: (args) => html`
+    <ic-checkbox-group label="Pick your coffee" name="group1">
+      <ic-checkbox value="valueName1" label="Americano"></ic-checkbox>
+      <ic-checkbox value="valueName2" checked>
+        <span slot="label">
+          Coffee <ic-status-tag label="hot" status="warning"></ic-status-tag>
+        </span>
+      </ic-checkbox>
+      <ic-checkbox value="valueName3" checked>
+        <span slot="label">
+          <b>Slot Coffee</b>
+        </span>
+      </ic-checkbox>
+    </ic-checkbox-group>
+  `,
+
+  name: "Slotted Label",
```

Comment:
It would be good if this was in lowercase to match the other checkbox story names - sorry, I know this is really nitpicky 😅 
```suggestion
  name: "Slotted label",
```